runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -47,6 +47,7 @@ template <typename LoaderDomain> class BaseLoader
   using Buffer = typename LoaderDomain::Buffer;
   using BuiltinOperator = typename LoaderDomain::BuiltinOperator;
   using CustomOptionsFormat = typename LoaderDomain::CustomOptionsFormat;
+  using Metadata = typename LoaderDomain::Metadata;
```

Comment:
Since both `tflite` and `circle` have metadata, Let add `Metadata` and `loadMetadata` in base_loader.

tflite_schema.fbs
https://github.com/Samsung/ONE/blob/11620233e1a303dcc60f48fc7103cb55d84270cc/runtime/onert/frontend/tflite/tflite_schema.fbs#L1239-L1244 

circle_schema.fbs (v0.7) 
https://github.com/Samsung/ONE/blob/11620233e1a303dcc60f48fc7103cb55d84270cc/res/CircleSchema/0.7/circle_schema.fbs#L1616-L1621